Sources/Mantis/CropView/CropView.swift

```diff
@@ -76,9 +79,10 @@ class CropView: UIView {
         print("CropView deinit.")
     }
     
-    init(image: UIImage, viewModel: CropViewModel = CropViewModel()) {
+    init(image: UIImage, viewModel: CropViewModel = CropViewModel(), config: Mantis.Config = Mantis.Config()) {
```

Comment:
Instead of passing Mantis.Config, can you just pass a DialConfig to CropView.init because CropView does not need the whole Config? Also we don't need a default value for it because we want to keep the config consistent.

Reply:
It needs the Config.rotationLimit...

Reply:
I think you can build DailConfig outside CropView with Config.rotationLimit, then pass it to CropView.